packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -304,6 +294,7 @@ const App = (props) => {
                 defaultLocale={DEFAULT_LOCALE}
             >
                 <CurrencyProvider currency={currency}>
+                    <StorefrontPreview />
```

Comment:
Why is this nested inside providers that it doesn't seem to use, rather than just within the component root?

Reply:
It does not really matter in this case. We can put it here or within component root. I prefer to put it here since it is not a Wrapped Component.